CODIGO/mDx8_Engine.bas

```diff
@@ -729,7 +729,80 @@ Public Sub Engine_Update_FPS()
        ' End If
 End Sub
 
+Public Sub DrawPJ(ByVal Index As Byte)
+    Dim cColor As Long
+   
+    If cPJ(Index).Nombre <> "" Then
+        frmPanelAccount.lblAccData(Index).Caption = cPJ(Index).Nombre
+        Select Case cPJ(Index).Color
+            Case 1 'Verde
+                cColor = RGB(0, 225, 0)
+            Case 2 'Azul
+                cColor = RGB(0, 0, 255)
+            Case 3 'Rojo
+                cColor = RGB(255, 0, 0)
+            Case 4 'Noble
+                cColor = RGB(207, 146, 3)
+            Case 5 'blanco
+                cColor = RGB(255, 255, 255)
+            
+        End Select
+        frmPanelAccount.lblAccData(Index).ForeColor = cColor
+    End If
 
+    If cPJ(Index).Nombre = "" Then Exit Sub
+   
+    Dim i As Integer
+   
+    Dim init_x As Integer
+    Dim init_y As Integer
+    Dim head_offset As Byte
+    Dim grhtemp As Grh
+    Static re As RECT
+   
+    re.Left = 0
+    re.Top = 0
+    re.bottom = 80
+    re.Right = 76
+   
+    init_x = 25
+    init_y = 20
+   
+    Dim Light(3) As Long
+    Light(0) = D3DColorXRGB(255, 255, 255)
+    Light(1) = D3DColorXRGB(255, 255, 255)
+    Light(2) = D3DColorXRGB(255, 255, 255)
+    Light(3) = D3DColorXRGB(255, 255, 255)
+    
+    If cPJ(Index).Race = eRaza.Humano Or cPJ(Index).Race = eRaza.Elfo Or cPJ(Index).Race = eRaza.ElfoOscuro Then
+        head_offset = 8
```

Comment:
what is the 8? can we create a constant to understand the meaning of the 8 ?

Reply:
That's the head offset, yes we can create a constant